### PR TITLE
Refactor FXIOS-7389 [v119] Update layout and copy for "No analysis" card

### PR DESCRIPTION
**Pull request by thatswinnie**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-7389)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/16379)

## :bulb: Description
Updates the layout, copy and image of the "No analysis" card and also displays it when the product was never analyzed.

Simulator Screenshot - iPhone 14 Pro - 2023-09-21 at 15 40 43

Simulator Screenshot - iPhone 14 Pro - 2023-09-21 at 16 13 37

## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [x] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed I updated documentation / comments for complex code and public methods

### CODE REVIEW

**Comment by razvanlitianu** [comment]
The image looks a bit pixelated doesn't it?

**Comment by thatswinnie** [comment]
@razvanlitianu that's by design - so it's correct like this.

**Review by razvanlitianu — APPROVED** [review]

File: Client/Assets/Images.xcassets/Shopping/shoppingNoAnalysisImage.imageset/Contents.json [hunk 1]
```diff
@@ -1,7 +1,17 @@
 {
   "images" : [
     {
-      "filename" : "shoppingNoAnalysisImage.pdf",
+      "filename" : "mobile iOS no-analysis 1x 280x108 light 72dpi MARGIN.png",
+      "idiom" : "universal"
```
Comment (razvanlitianu) [inline Client/Assets/Images.xcassets/Shopping/shoppingNoAnalysisImage.imageset/Contents.json:5]: I know it was mentioned the image is by design pixelated but I see the naming contains 1x. Shouldn't we have 2 and 3x. variants as well?

Comment (thatswinnie) [reply Client/Assets/Images.xcassets/Shopping/shoppingNoAnalysisImage.imageset/Contents.json:5]: I showed design the screenshots with the 1x images and they approved the usage like this.

**Review by yoanarios — APPROVED** [review]

**Pull request merged**